<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>262</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_transform">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать шаблон</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_doc">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>43</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Вид документа:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_type_doc">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>40</y>
      <width>185</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_select_doc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Выбрать документ</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_doc_name">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>9</y>
      <width>275</width>
      <height>24</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_add_data">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>76</y>
      <width>262</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>  Добавить данные в шаблонный документ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_faculty">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>341</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_faculty">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>113</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Факультет:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_department">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>143</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кафедра:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_department">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>341</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_theme">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>341</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_theme">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>174</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тема:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_student">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>203</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Студент:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_headOfWork">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>203</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Рук. работы:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_headOfWork">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>199</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_student">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>115</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_headOfDepartment">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>231</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Зав. кафедрой:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_headOfDepartment">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>227</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_reviewer">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>227</y>
      <width>115</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_reviewer">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Рецензент:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
